contract-map.json

```diff
@@ -4527,5 +4527,12 @@
     "erc20": true,
     "symbol": "PKF",
     "decimals": 18
+  },
+  "0x06325440D014e39736583c165C2963BA99fAf14E": {
+    "name": "Curve.fi",
+    "logo": "curve.svg",
```

Comment:
This logo is slightly different than the one you sent me. The logo in the PR is just the regular CRV token image, but the one you sent me is the specific logo for steCRV

Reply:
I did not find a good svg one but I'll add the one I showed you for consistency. Thanks for reviewing!